internal/fsutil/create_test.go

```diff
@@ -86,13 +86,74 @@ var createTests = []createTest{{
 		Mode: 0644,
 		Data: bytes.NewBufferString("changed"),
 	},
-	hackdir: func(c *C, dir string) {
+	hackopt: func(c *C, dir string, options *fsutil.CreateOptions) {
 		c.Assert(os.WriteFile(filepath.Join(dir, "foo"), []byte("data"), 0666), IsNil)
 	},
 	result: map[string]string{
 		// mode is not updated.
 		"/foo": "file 0666 d67e2e94",
 	},
+}, {
+	// Create a hard link to `file`
```

Comment:
Let's do this properly once and for all :D, can you introduce a `summary` field and describe the tests (the new and the old ones)?